.travis.yml

```diff
@@ -32,9 +32,25 @@ before_script:
 # Package installation
 install:
   - pip install codecov
+
   # Install project dependencies
   - pip install -r requirements.txt
 
+  # Install node
+  - nvm install 8
+
+  # Move into the static_src folder where we will compile the FE
+  - cd ./opentech/static_src
+
+  # Install node dependencies
+  - npm install --quiet
+
+  # Build the static files
+  - npm run build:prod
```

Comment:
what does that accomplish on Travis? we are not doing JS/visual regression tests

Reply:
Currently it will confirm that it compiles successfully. 

We should probably add the collectstatic as part of the test phase to ensure we have all the static files available.